src/client/opamCommands.ml

```diff
@@ -226,9 +226,15 @@ let init =
     in
     let init_config =
       try
-        OpamConsole.note "Will configure from built-in defaults%s."
-          (OpamStd.List.concat_map ~nil:"" ~left:", " ", "
-             OpamFile.to_string config_files);
+        let others =
+          match config_files with
+          | [] -> ""
+          | [file] -> OpamFile.to_string file ^ " and then from "
+          | _ ->
+              (OpamStd.List.concat_map ~nil:"" ~right:", and finally from " ", then "
```

Comment:
Didn't you mean to use `~last_sep` here ?

Reply:
@dra27 ping ?

Reply:
Ah, no, nevermind ^^

Reply:
Sorry :)